src/Abp/Application/Features/Feature.cs

```diff
@@ -93,7 +91,7 @@ public IReadOnlyList<Feature> Children
         /// <param name="inputType">Input type</param>
         public Feature(string name, string defaultValue, ILocalizableString displayName = null, ILocalizableString description = null, FeatureScopes scope = FeatureScopes.All, IInputType inputType = null)
         {
-            if (name == null)
+            if (name.IsNullOrEmpty())
```

Comment:
This seems unnecessary. And it may not be appropriate to throw a `null` exception for an empty string.

[L91](https://github.com/aspnetboilerplate/aspnetboilerplate/pull/2978/files#diff-3a7102fb563aa4d80bf6af24f267252aL91): "A brief description for `the` feature"

Reply:
What do you propose instead?

Reply:
Name = name ?? throw new ArgumentNullException("name");

Reply:
Visual Studio does suggest that, but I'm not sure if that's what @hikalkan wants.

Reply:
Ok, I will wait to see what @hikalkan says.